Kernel/FileSystem/FATFS/Inode.cpp

```diff
@@ -186,14 +186,6 @@ ErrorOr<NonnullOwnPtr<KString>> FATInode::compute_filename(FATEntry& entry, Vect
     VERIFY_NOT_REACHED();
 }
 
-Time FATInode::fat_date_time(FATPackedDate date, FATPackedTime time)
-{
-    if (date.value == 0)
-        return Time();
-
-    return Time::from_timestamp(first_fat_year + date.year, date.month, date.day, time.hour, time.minute, time.second * 2, 0);
```

Comment:
`first_fat_year` seems to be unused now. However, I'd onsider to carry that over to the AK implementation, replacing the `1980` magic number.